components/event-bus/cmd/event-bus-subscription-controller-knative/cmd/manager/main.go

```diff
@@ -0,0 +1,132 @@
+/*
+Copyright 2019 The Kyma Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
+	"github.com/kyma-project/kyma/components/event-bus/cmd/event-bus-subscription-controller-knative/pkg/webhook"
+	"github.com/kyma-project/kyma/components/event-bus/internal/common"
+	"github.com/kyma-project/kyma/components/event-bus/internal/knative/subscription/controller"
+	"github.com/kyma-project/kyma/components/event-bus/internal/sck/opts"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+)
+
+func main() {
+	sckOpts := opts.ParseFlags()
+
+	logf.SetLogger(logf.ZapLogger(false))
+	log := logf.Log.WithName("entrypoint")
```

Comment:
Please use a different name other than `log` as it will collide with the imported `log` package.

Reply:
There were two different loggers being used. I deleted one of them now, so it should be ok to use `log`.